pkgs/development/python-modules/fastavro/default.nix

```
@@ -0,0 +1,51 @@
+{ buildPythonPackage
+, cython
+, dateutil
+, fetchFromGitHub
+, lib
+, lz4
+, numpy
+, pandas
+, pytestCheckHook
+, python-snappy
+, zstandard
+}:
+
+buildPythonPackage rec {
+  pname = "fastavro";
+  version = "1.4.4";
+
```

```suggestion
  version = "1.4.4";

    disabled = pythonOlder "3.6";
```

see https://github.com/fastavro/fastavro/blob/master/setup.py#L98

---

Thanks!